packages/modal/addon/components/nucleus-modal.js

```diff
@@ -242,7 +242,9 @@ class Modal extends Component {
   *
   */
   attachEventHandlers() {
-    window.addEventListener("focusin", event => this.loopFocus(event));
+    let focusListener = this.loopFocus.bind(this);
+    set(this, "_focusListener", focusListener);
+    window.addEventListener("focusin", focusListener);
```

Comment:
Nice 🥇

Reply:
Can you move the focus handling part as a separate mixin? Something like https://github.com/freshdesk/nucleus/blob/master/packages/modal/addon/mixins/scroll.js.